### PR TITLE
NAS documentation

### PR DESCRIPTION
**Pull request by camenpihor**
works on #32 
works on #30 
works on #31 
works on #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+1. For detailed documentation on a number of topics and bugs at NAS please see [here.](https://www.nas.nasa.gov/hecc/support/kb/)
+1. For an overview of many common NAS tasks see [here.](https://www.nas.nasa.gov/hecc/support/kb/basic-tasks_264.html)
+1. For an overview of NAS Compute Architecture see [here.](https://www.nas.nasa.gov/hecc/support/kb/hpc-environment-overview_25.html)
+1. Support can be reached easily by phone at 800-331-8737 and <support@nas.nasa.gov>.
+
+### Storage
+
+1. All data should be stored on Lou. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/the-lou-mass-storage-system_371.html).
+1. For all data storage purposes, please store your data on ` Lou:/u/your_nas_username`
+1. `/nobackup/your_nas_username` is cleared regularly regularly. `home4/your_nas_username` is not.
```
Comment (camenpihor) [inline README.md]:
```suggestion
1. `/nobackup/your_nas_username` is cleared regularly. `home4/your_nas_username` is not.
```

File: README.md [hunk 2]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+1. NAS has much of the GOES16 data we are working with available in `nex/datapool/goes16`, accessible from pfe.
+1. THe `/nasa` folder available via pfe has a lot of useful software you can easily load.
```
Comment (camenpihor) [inline README.md]:
```suggestion
1. The `/nasa` folder available via pfe has a lot of useful software you can easily load.
```

File: README.md [hunk 3]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
 ## goes/
 
 Utilties for using NOAA's GOES-16/17 satellite data, which is publicly accessible in Amazon S3.
+
+## NAS
+
+This project is meant to be deployed both locally and on NASA's NEX Super Computers.
+1. For detailed documentation on a number of topics and bugs at NAS please see [here.](https://www.nas.nasa.gov/hecc/support/kb/)
+1. For an overview of many common NAS tasks see [here.](https://www.nas.nasa.gov/hecc/support/kb/basic-tasks_264.html)
```
Comment (camenpihor) [inline README.md]:
```suggestion
1. For an overview of many common NAS tasks see [here](https://www.nas.nasa.gov/hecc/support/kb/basic-tasks_264.html).
```

File: README.md [hunk 4]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
 ## goes/
 
 Utilties for using NOAA's GOES-16/17 satellite data, which is publicly accessible in Amazon S3.
+
+## NAS
+
+This project is meant to be deployed both locally and on NASA's NEX Super Computers.
+1. For detailed documentation on a number of topics and bugs at NAS please see [here.](https://www.nas.nasa.gov/hecc/support/kb/)
```
Comment (camenpihor) [inline README.md]:
```suggestion
1. For detailed documentation on a number of topics and bugs at NAS please see [here](https://www.nas.nasa.gov/hecc/support/kb/).
```

File: README.md [hunk 5]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+1. For an overview of many common NAS tasks see [here.](https://www.nas.nasa.gov/hecc/support/kb/basic-tasks_264.html)
+1. For an overview of NAS Compute Architecture see [here.](https://www.nas.nasa.gov/hecc/support/kb/hpc-environment-overview_25.html)
```
Comment (camenpihor) [inline README.md]:
```suggestion
1. For an overview of NAS Compute Architecture see [here](https://www.nas.nasa.gov/hecc/support/kb/hpc-environment-overview_25.html).
```

File: README.md [hunk 6]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+1. Support can be reached easily by phone at 800-331-8737 and <support@nas.nasa.gov>.
+
+### Storage
+
+1. All data should be stored on Lou. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/the-lou-mass-storage-system_371.html).
+1. For all data storage purposes, please store your data on ` Lou:/u/your_nas_username`
+1. `/nobackup/your_nas_username` is cleared regularly regularly. `home4/your_nas_username` is not.
+1. NAS has much of the GOES16 data we are working with available in `nex/datapool/goes16`, accessible from pfe.
```
Comment (camenpihor) [inline README.md]: Maybe expanding what PFE means, or a quick overview of the different compute servers?

File: README.md [hunk 7]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+
+1. Conda is natively supported on NAS. On a given instance, to ensure that conda is loaded, please call:
+`% module use -a /u/analytix/tools/modulefiles` and `% module load miniconda3/v4`.
+1. You can create your own conda environments easily, and call them on any interactive or reserved node.
+
+### Interactive Sessions
+
+All users of NAS can launch an interactive PBS session with qsub, with a sample command found here:
+`qsub -I -q v100 -l select=1:ncpus=1:model=sky_gpu,walltime=12:00:00`. The node ID (for instance `r101i0n4`) is constant 
+per user (it does not change per different interactive sessions).
```
Comment (camenpihor) [inline README.md]: when does the `node ID` appear?

File: README.md [hunk 8]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+
+Alternatively, users can launch jobs via a PBS script. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/using-conda-environments-for-machine-learning_557.html.
```
Comment (camenpihor) [inline README.md]:
```suggestion
Alternatively, users can launch jobs via a PBS script. For more information please see [here](https://www.nas.nasa.gov/hecc/support/kb/using-conda-environments-for-machine-learning_557.html).
```

File: README.md [hunk 9]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+A sample script can be found in the examples folder as well.
+
+### Reserved Notes
+
+All users can reserve a compute nodes. For more information see [here.](https://www.nas.nasa.gov/hecc/support/kb/reserving-a-dedicated-compute-node_556.html)
```
Comment (camenpihor) [inline README.md]:
```suggestion
All users can reserve a compute node. For more information see [here](https://www.nas.nasa.gov/hecc/support/kb/reserving-a-dedicated-compute-node_556.html).
```

File: README.md [hunk 10]
```diff
@@ -45,3 +46,61 @@ Checkout the `examples/` directory for example notebooks.
...
+These reserved nodes are nice ways to setup long-running Jupyter Notebooks. Users may have at most one reserved node.
```
Comment (camenpihor) [inline README.md]:
```suggestion
These reserved nodes are nice ways to set up long-running Jupyter Notebooks. Users may have at most one reserved node on the Pleaides compute cluster.
```

File: examples/sample_ssh_config [hunk 11]
```diff
@@ -0,0 +1,49 @@
+## For ALL ##
+Host *
+        KexAlgorithms +diffie-hellman-group1-sha1
+        Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
+
+## For NAS Updated 20190315 ##
+Host pfe pfe-last pfe.nas.nasa.gov pfe-last.nas.nasa.gov
+        ProxyCommand ssh shzimmer@sfe2.nas.nasa.gov /usr/local/bin/ssh-balance %h
```
Comment (camenpihor) [inline examples/sample_ssh_config]: I think you should remove your username and replace it with something like <NAS USER NAME>

File: README.md [hunk 12]
```diff
@@ -11,6 +11,7 @@ geostationary satellite imagery.
   - [Instructions](#Instructions)
   - [Requirements](#Requirements)
 - [goes/](#goes/)
+- [NAS](#NAS)
```
Comment (camenpihor) [inline README.md]: Maybe adding the subsections too? like `Storage`

**Comment by camenpihor** [comment]
It seems like by creating a PR that it is saying that I am the author of it? I tried various ways to tell github that you're the author to no avail. So, we could address the comments here, close this PR, and  you create a new one?

In general, I think creating "draft PRs" are a good strategy for requesting feedback on unfinished work, or like a sanity check on finished work.

**Comment by camenpihor** [comment]
Does this code work on or fix any other issues?

**Comment by camenpihor** [comment]
I think another thing that we could do, is decide if we want this documentation to read more like instructions or like tidbits of useful knowledge. I think right now it reads more like instructions, but we may want to link to their published instructions and then document weird or unintuitive things about it?

I don't think we have to do this, and we can always change it later, I just briefly thought about this, and so decided to make a note of it!

Thanks Sam! :)

File: README.md (hunk 12 above)
Comment (Ferrumofomega) [reply README.md]: Added!

**Comment by camenpihor** [comment]
I think that this is ready to merge except for  debug/test code for xarray and dask. So I am separating that stuff from this PR. 

THANKS FOR DOING ALL OF THIS! Tons of work here, and its awesome :) 

Creating a new PR for the xarray and dask code that you had. I'll link that stuff here

**Comment by camenpihor** [comment]
moved the dask/xarray stuff to https://github.com/joyprojects/wildfire/pull/53

**Pull request merged**